index.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable linebreak-style */
+class BookList {
+  constructor() {
+    this.bookList = [];
+  }
+
+  addBook(title = '', author = '') {
+    const list = document.querySelector('.booklist');
+    list.style.padding = '0';
+    const li = document.createElement('li');
+    li.style.margin = '0';
+    li.style.listStyleType = 'none';
+    const bookTitle = document.createElement('p');
+    bookTitle.className = 'title';
+    bookTitle.innerText = title;
+    const bookAuthor = document.createElement('p');
+    bookAuthor.className = 'author';
+    bookAuthor.innerText = author;
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'remove';
+    removeBtn.innerText = 'Remove';
+    const hr = document.createElement('hr');
+    hr.className = 'bottom-border';
+    li.appendChild(bookTitle);
+    li.appendChild(bookAuthor);
+    li.appendChild(removeBtn);
+    li.appendChild(hr);
+    list.appendChild(li);
+    const book = { title, author };
+    this.bookList.push(book);
+    window.localStorage.setItem('bookList', JSON.stringify(this.bookList));
+  }
+
+  removeBook(index) {
+    this.bookList = this.bookList.filter((book, i) => this.bookList[index] !== this.bookList[i]);
+    window.localStorage.setItem('bookList', JSON.stringify(this.bookList));
+  }
+}
```

Comment:
- [ ] Please, remove this class for this milestone.  The reason is that in the next milestone, you will refactor this code to classes.  Kindly ensure that in this milestone your project does not depend on any class, use arrays and objects instead.

_Hint: You could save this code somewhere and use it for the next milestone, so your work is preserved.  Nevertheless, I commend you for your extra effort 👏_

Reply:
I am really grateful for the hints.

Reply:
💪